engine/esma_template.c

```diff
@@ -297,6 +297,19 @@ static int __decode_dbuf(char *line, struct esma_template *et)
 	s = p;
 	while (p != end) {
 
+		if ('/' == *p && '/' == *(p + 1)) {
+			while (*p != '\n')
+				p++;
+		}
```

Comment:
If the commented line is the last one and it does not end with the newline symbol, we'll get off the buffer read, correct?